### DISCUSSION TITLE
skywalking ui does not work

### DISCUSSION THREAD

**Discussion by hieu276** [apache/skywalking#9683] [Q&A]
Skywalking APM version 9.2.0
OS Ubuntu 22.04.1 LTS
I'm facing this error when running skywalking ui from script bin/startup.sh.
![image](https://user-images.githubusercontent.com/48582387/192175196-6bda6c6a-cac7-4d1a-9ac8-5710b7ae3d04.png)

**Comment by luojie233** [comment]
Hi, I have the same problem, did you resolve it?

**Reply by hieu276** [reply]
@luojie233  No, i didn't. You can try run it on another OS such as ubuntu 20.04, centOS 8. It works for me.

**Reply by luojie233** [reply]
Thanks!
